src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -137,6 +139,37 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchCurrentUser(currentUser);
     }
 
+    componentDidUpdate({ annotatorState: prevAnnotatorState }: Props): void {
+        const { error: prevError } = prevAnnotatorState;
+        const {
+            annotatorState: { error },
+        } = this.props;
+
+        if (prevError !== error && error) {
+            switch (error.code) {
+                case ERROR_CODE_FETCH_ANNOTATIONS:
+                    this.removeAnnotations();
```

Comment:
Maybe `filterAnnotations` to be consistent?

Reply:
There's already one `filterAnnotations` which is called by `removeAnnotations`.